bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/CallOptionsFactory.java

```diff
@@ -47,4 +53,27 @@
       return CallOptions.DEFAULT;
     }
   }
+
+  /**
+   * Creates a new {@link CallOptions} based on a {@link CallOptionsConfig}. This factory assumes
+   * {@link CallOptionsConfig#isUseTimeout()} is true.
```

Comment:
Why does it assume isUseTimeout == true? It seems to handle both cases in create().